CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - New check in RetrieveBatchSecretSafe method which will return an error if the `Content-Type` header
   is not set in the response (this indicates Conjur is out of date with the client).
   [cyberark/conjur-api-go#104](https://github.com/cyberark/conjur-api-go/issues/104)
+- ResourcesRequest is now supported for v4 conjur instances.
```

Comment:
```suggestion
- ResourcesRequest is now supported for Conjur Enterprise v4.
  [PR cyberark/conjur-api-go#106](https://github.com/cyberark/conjur-api-go/pull/106)
```

I am not sure whether it's worth including this since our next release soon will deprecate v4, so I'd also be ok with deleting it. up to you